### PR TITLE
Implementation WeatherForecast

### PR DESCRIPTION
**Pull request by fdelcampo**

### CODE REVIEW

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]
Great job @fdelcampo ! I really appreciate you improved and refactored several things. I left comments regarding code quality and a few suggestion, please take a look to them 🙏️

File: love/src/components/icons/Weather/CloudInSkyIcon/CloudInSkyIcon.jsx
[the remark this answers is not in the document]
Comment (fdelcampo) [reply love/src/components/icons/Weather/CloudInSkyIcon/CloudInSkyIcon.jsx:1]: In this moment not used with a refactor. This file removed

File: love/src/components/GeneralPurpose/Plot/TimeSeriesControls/TimeSeriesControls.jsx [hunk 1]
```diff
@@ -47,29 +48,12 @@ export default class TimeSeriesControls extends Component {
...
-            className={[styles.modeSelection, !this.props.isLive ? styles.highlightText : ''].join(' ')}
-            onClick={() => this.props.setLiveMode(false)}
-          >
-            Query
-          </span>
+          <Toggle
+            labels={['Live', 'Query']}
+            isLive={!this.props.isLive}
+            setLiveMode={(show) => this.props.setLiveMode(!show)}
+          />
```
[the remark this answers is not in the document]
Comment (fdelcampo) [reply love/src/components/GeneralPurpose/Plot/TimeSeriesControls/TimeSeriesControls.jsx:55]: It's necessary for the actual comportament, the negation operation

**Review by sebastian-aranda — APPROVED** [review]
Looks good to be merged :+1:

File: love/src/components/WeatherStation/CurrentValues/CurrentValues.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    currentWindSpeed: PropTypes.number,
+    currentWindSpeedUnits: PropTypes.string,
+  };
+
+  static defaultProps = {
+    currentTemperature: undefined,
+    currentHumidity: undefined,
+    currentPressure: undefined,
+    currentWindSpeed: undefined,
+    currentWindSpeedUnits: undefined,
```
[the remark this answers is not in the document]
Comment (fdelcampo) [reply love/src/components/WeatherStation/CurrentValues/CurrentValues.jsx:25]: In the case that is `undefined`, the text is `'-'` and not a `'number °C'`, for example

**Pull request merged**